blackbox/docs/sql/reference/select.txt

```diff
@@ -31,7 +31,9 @@ SELECT retrieves rows from a table. The general processing of SELECT
 is as follows:
 
 - The FROM item points to the table where the data should be retrieved
-  from.
+  from. If no FROM item is specified, the query is instead executed against
+  ``sys.cluster``. This is so that queries that are commonly used in connection
```

Comment:
Imho this shouldn't be the case. `select * from sys.cluster` should work. But `select *` should raise an error.